src/components/dropdown/README.md

```diff
@@ -1 +1,138 @@
+  <div class="rvt-dropdown__menu-heading" aria-hidden="true">Menu list heading</div>
+    <div role="group" aria-label="Related">
+        <a href="#">Nested item one</a>
+        <a href="#">Nested item two</a>
+    </div>
+  </div>
+</div>
+```
+
+## Initialization
+
+In order to initialize the dropdown as an ES6 module, you will need to create a new instance of it, and pass it the element you wish to instantiate as a dropdown.
+
+```
+const dropdownElement = document.querySelector('[data-dropdown="your-data-dropdown-value"]');
+document.newDropdown = new Rivet.Dropdown(dropdownElement);
+```
+
+## "Public" methods
+
+If you use the appropriate data attribute/id combination in your markup, the dropdown will work without the need for any additional JavaScript. But if you need to control the dropdown programmatically, there are several methods from the Rivet dropdown's API you can use:
+
+| Method           | Description                                                                            |
+| ---------------- | -------------------------------------------------------------------------------------- |
+| .open()  | Opens the dropdown menu list |
+| .close() | Closes the dropdown menu list |
+| .init()  | Adds the built-in event listeners to the dropdown |
+| .destroy() | Removes all built-in event listeners from the dropdown |
+
+## Custom events
+
+| Event        | Description                                                                                                                                                                                                                                                          |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `dropdownOpen`  | Emitted when the dropdown is opened (using the Dropdown.open() method). The value of the dropdown id attribute is also passed along via the custom event’s detail property and is available to use in your scripts as event.detail.id()  |
+| `dropdownClose` | Emitted when the dropdown is closed (using the Dropdown.close() method). The value of the dropdown id attribute is also passed along via the custom event’s detail property and is available to use in your scripts as event.detail.id() |
+
+## Accessibility requirements
+
+The Rivet dropdown is built to follow the WAI-ARIA authoring standards. It is marked up with the appropriate ARIA attributes and uses the JavaScript included in `rivet.js` to implement the keyboard navigation and focus management required to meet the [ARIA Authoring Practices standards](http://w3c.github.io/aria-practices/). If you need to create the dropdown functionality in another framework/library like React, Angular, etc., please ensure that it meets the following accessibility requirements.
```

Comment:
This is a small detail, but since we now ship multiple versions of Rivet JS, you might need to update some language here. For example, we don't ship a JS file called `rivet.js` anymore since there is an ESM and IIFE version.